modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -781,17 +781,20 @@ protected function render() {
 		$items_title_html = '';
 		$icons_content = $this->render_accordion_icons( $settings );
 		$this->add_render_attribute( 'elementor-accordion', 'class', 'e-n-accordion' );
+		$this->add_render_attribute( 'elementor-accordion', 'aria-label', 'Accordion' );
 		$default_state = $settings['default_state'];
 		$title_html_tag = Utils::validate_html_tag( $settings['title_tag'] );
 
 		$faq_schema = [];
 
 		foreach ( $items as $index => $item ) {
+			$accordion_count = $index + 1;
 			$item_setting_key = $this->get_repeater_setting_key( 'item_title', 'items', $index );
 			$item_classes = [ 'e-n-accordion-item', 'e-normal' ];
 			$item_id = empty( $item['element_css_id'] ) ? 'e-n-accordion-item-' . $id_int . $index : $item['element_css_id'];
 			$item_title = $item['item_title'];
 			$is_open = 'expanded' === $default_state && 0 === $index ? 'open' : '';
+			$aria_expanded = 'expanded' === $default_state && 0 === $index ? 'true' : 'false';
```

Comment:
I wonder what is the logic here, but I can see that you got it from the line above.